vnext/Microsoft.ReactNative.Managed/JSValue.cs

```diff
@@ -254,7 +254,7 @@ public override int GetHashCode()
     // It does not implement GetHashCode() and must not be used as a key in a dictionary.
     public struct JSValue : IEquatable<JSValue>
     {
-        public static readonly JSValue Null = new JSValue();
+        public static readonly JSValue Null = default;
```

Comment:
>default [](http://example.com/codeflow?start=46&length=7)

Isn't this already done if you leave it out?

Reply:
No, there was a compilation error where they wanted to have a constant variable with static initialization, while calling the constructor is a dynamic initialization. The result value is exactly the same, but the "default" one could be used as a default value for optional function parameters, while the `new JSValue()` cannot be used for that.